`gpu_nvidia` module cannot build when enabled

*   Version of collectd:
master
*   Operating system / distribution:
Ubuntu 16.04
*   Kernel version (if applicable):

## Expected behavior

(Description of the behavior / output that you expected)
Build gpu_nvidia successfully with 
```
./configure --enable-gpu-nvidia --with-cuda=/usr/local/cuda/
```

## Actual behavior

(Description of the behavior / output that you observed)
- could not find `collectd.h` when compiling `gpu_nvidia.c`

## Steps to reproduce

*   step 1
```
./configure --enable-gpu-nvidia --with-cuda=/usr/local/cuda/
```
*   step 2
```make```
*   step 3

Here is what I see from `master` branch:
```
$ make
...
  CCLD     email.la
  CC       src/entropy.lo
  CCLD     entropy.la
  CC       src/exec.lo
  CCLD     exec.la
  CC       src/ethstat.lo
  CCLD     ethstat.la
  CC       src/fhcount.lo
  CCLD     fhcount.la
  CC       src/filecount.lo
  CCLD     filecount.la
  CC       src/fscache.lo
  CCLD     fscache.la
  CC       src/gpu_nvidia_la-gpu_nvidia.lo
In file included from src/daemon/plugin.h:34:0,
                 from src/gpu_nvidia.c:24:
./src/utils/metadata/meta_data.h:30:10: fatal error: collectd.h: No such file or directory
 #include "collectd.h"
          ^~~~~~~~~~~~
compilation terminated.
Makefile:6954: recipe for target 'src/gpu_nvidia_la-gpu_nvidia.lo' failed
make[1]: *** [src/gpu_nvidia_la-gpu_nvidia.lo] Error 1
make[1]: Leaving directory '/home/guanghui/projects/collectd'
Makefile:4727: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -1017,7 +1017,7 @@ endif
 if BUILD_PLUGIN_GPU_NVIDIA
 pkglib_LTLIBRARIES += gpu_nvidia.la
 gpu_nvidia_la_SOURCES = src/gpu_nvidia.c
-gpu_nvidia_la_CPPFLAGS = $(PLUGIN_CPPFLAGS) $(BUILD_WITH_GPU_CUDA_CPPFLAGS)
+gpu_nvidia_la_CPPFLAGS = $(AM_CPPFLAGS) $(BUILD_WITH_GPU_CUDA_CPPFLAGS)
 gpu_nvidia_la_LDFLAGS = $(PLUGIN_LDFLAGS) $(BUILD_WITH_GPU_CUDA_LDFLAGS)
 gpu_nvidia_la_LIBADD = $(BUILD_WITH_CUDA_LIBS)
 endif

